src/it/mangaworld/src/eu/kanade/tachiyomi/extension/it/mangaworld/Mangaworld.kt

```diff
@@ -150,16 +150,16 @@ class Mangaworld : ParsedHttpSource() {
         val metaData = document.select("div.comic-info").first()
 
         val manga = SManga.create()
-        manga.author = infoElement.select("a[href^=https://www.mangaworld.io/archive?author=]").first()?.text()
-        manga.artist = infoElement.select("a[href^=https://www.mangaworld.io/archive?artist=]")?.text()
+        manga.author = infoElement.select("a[href^=https://www.mangaworld.in/archive?author=]").first()?.text()
+        manga.artist = infoElement.select("a[href^=https://www.mangaworld.in/archive?artist=]")?.text()
```

Comment:
```suggestion
        manga.artist = infoElement.select("a[href^=$baseUrl/archive?artist=]")?.text()
```

Replace in the other ones that are harcoded too.